YoutubeExplode/Videos/VideoController.cs

```diff
@@ -174,8 +172,10 @@ public async ValueTask<PlayerResponse> GetPlayerResponseAsync(
               "videoId": {{Json.Serialize(videoId)}},
               "context": {
                 "client": {
-                  "clientName": "TVHTML5_SIMPLY_EMBEDDED_PLAYER",
-                  "clientVersion": "2.0",
+                  "clientName": "ANDROID",
+                  "clientVersion": "20.10.38",
+                  "osName": "Android",
+                  "osVersion": "11",
```

Comment:
This needs to remain the same because it needs the `TVHTML5_SIMPLY_EMBEDDED_PLAYER` to work around age-restricted videos

Reply:
done